teachertool/src/components/CatalogOverlay.tsx

```diff
@@ -40,26 +40,28 @@ interface CatalogItemLabelProps {
     recentlyAdded: boolean;
 }
 const CatalogItemLabel: React.FC<CatalogItemLabelProps> = ({ catalogCriteria, isMaxed, recentlyAdded }) => {
-    const showRecentlyAddedIndicator = recentlyAdded && !isMaxed;
     return (
         <div className={css["catalog-item-label"]}>
             <div className={css["action-indicators"]}>
                 {isMaxed ? (
-                    <span>{Strings.Max}</span>
+                    <i
+                        className="fas fa-check"
+                        title={Strings.MaxReached}
+                    />
                 ) : (
                     <>
                         <i
                             className={classList(
                                 "fas fa-check",
                                 css["recently-added-indicator"],
-                                showRecentlyAddedIndicator ? undefined : css["hide-indicator"]
+                                recentlyAdded ? undefined : css["hide-indicator"]
```

Comment:
I'm confused as to what this is doing if we already have the check mark above.

Reply:
The check mark above is only shown in the "maxed" scenario and has slightly different styling. The check mark here is traded off with the + button when a criteria is not maxed (it's what you see when there's a temporary green check that swaps out with the plus sign).

We could probably combine them into one single check mark component, but all we'd be saving there is lines of code, and I think it'd make things more confusing overall trying to reason about maxed and/or recently added and different styles in each scenario, etc...so I just kept them separate.